workspace.json

```diff
@@ -3649,7 +3715,7 @@
           "builder": "@nrwl/workspace:run-commands",
           "options": {
             "commands": [
-              "curl https://api.syslumenn.is/Test/swagger/v1/swagger.json -H 'Accept: application/json' > src/clientConfig.json",
+              "curl https://api.syslumenn.is/dev/swagger/v1/swagger.json -H 'Accept: application/json' > src/clientConfig.json",
```

Comment:
Because Styrmir from Sýslumenn had not updated the Test API so it defined the response schema correctly for the endpoint /VedbokavottordRegluverk.

But I will change this back and ask Styrmir to update their Test service asap

Reply:
I see, guessed that it might be the reason. But I think we should try to not depend on third party dev env as it might not be as stable as their test env.